### DISCUSSION TITLE
Round up calculation results

### DISCUSSION THREAD

**Discussion by vincentx1908** [MatthewJRobertson94/Tafe230132#3] [Ideas]
I'm thinking about rounding up the results for my currency converters to two decimal numbers, is it a good idea?

**Comment by MatthewJRobertson94** [comment]
That sounds good and would improve readability and practicality.

**Comment by MatthewJRobertson94** [comment]
I need your public key before you upload

[cid:d774ccbe-a989-4ad5-a173-5280d378e06c]
________________________________
From: Vincent Xiong ***@***.***>
Sent: 28 March 2023 14:14
To: MrTafeStudentPleasePassMe/Tafe230132 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [MrTafeStudentPleasePassMe/Tafe230132] Round up calculation results (Discussion #3)


I'm thinking about rounding up the results for my currency converters to two decimal numbers, is it a good idea?
